### PR TITLE
Content update

### PR DESCRIPTION
**Pull request by rutavshah**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -32,10 +32,11 @@ Learn how to use MicroShed Testing to test a MicroProfile or Jakarta EE applicat
 == What you'll learn
 
 You'll start with an existing REST application that runs on Open Liberty and use https://microshed.org/microshed-testing/[MicroShed Testing^] 
-to write tests for the application that exercise the application inside of a Docker container.
+to write tests for the application that exercise the application in a Docker container.
 
-Sometimes tests might pass in development and testing (dev/test) environments, but fail in production because the application is
-running differently in production than it is in dev/test. Fortunately, you can minimize these parity issues between development and production
+Sometimes tests might pass in development and testing (dev/test) environments,
```
Comment (AustinSeto) [inline README.adoc:37]: "testing (dev/test)" seems redundant, can we just say "testing environments"?

File: README.adoc [hunk 2]
```diff
@@ -154,54 +157,35 @@ Wait for the `Press the Enter key to run tests on demand.` message, and then pre
...
 ----
 
-The [hotspot=withReadinessPath file=1]`withReadinessPath(String)` method indicates what path is polled by HTTP to determine application readiness. 
+The [hotspot=withReadinessPath file=0]`withReadinessPath(String)` method indicates what path is polled by HTTP to determine application readiness. 
 MicroShed Testing automatically starts the ApplicationContainer application and waits for it to be ready before the tests start running. 
-In this case, you are using the default application readiness check at the http://localhost:9080/health/ready[http://localhost:9080/health/ready^] URL, which is enabled by the [hotspot=mpHealth file=2]`MicroProfile Health` feature in our server.xml configuration file. When the readiness URL returns `HTTP 200`, the application is considered ready and the tests begin running.
+In this case, you're using the default application readiness check at the http://localhost:9080/health/ready[http://localhost:9080/health/ready^] URL, which is enabled by the [hotspot=mpHealth file=1]`MicroProfile Health` feature in our server.xml configuration file. When the readiness URL returns `HTTP 200`, the application is considered ready and the tests begin running.
```
Comment (AustinSeto) [inline README.adoc]: Update `MicroProfile Health` to `mpHealth-3.0`

**Review by AustinSeto — CHANGES_REQUESTED** [review]
Just some minor suggestions for the README content.

Other than that, the static version of the guide looks guide, instructions and app all work.

File: README.adoc (hunk 2 above)
Comment (gkwan-ibm) [reply README.adoc]: No need to update at here because not want to hard code the version 3.0

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc:37]: Let ID to review. Line 38 is using dev/test.

File: README.adoc [hunk 3]
```diff
@@ -32,10 +32,11 @@ Learn how to use MicroShed Testing to test a MicroProfile or Jakarta EE applicat
 == What you'll learn
 
 You'll start with an existing REST application that runs on Open Liberty and use https://microshed.org/microshed-testing/[MicroShed Testing^] 
-to write tests for the application that exercise the application inside of a Docker container.
+to write tests for the application that exercise the application in a Docker container.
 
-Sometimes tests might pass in development and testing (dev/test) environments, but fail in production because the application is
-running differently in production than it is in dev/test. Fortunately, you can minimize these parity issues between development and production
+Sometimes tests might pass in development and testing (dev/test) environments, 
+but fail in production because the application is running differently in production than it's in dev/test.
```
Comment (Rwalls1) [inline README.adoc]: Change "is running differently" to "runs differently". Also, change "than it's in dev/test" to "than in dev/test".

File: README.adoc [hunk 4]
```diff
@@ -32,10 +32,11 @@ Learn how to use MicroShed Testing to test a MicroProfile or Jakarta EE applicat
...
+Fortunately, you can minimize these parity issues between development and production
```
Comment (Rwalls1) [inline README.adoc]: Change "parity issues" to "differences". Also, change "development" to "dev/test" to maintain consistency.

File: README.adoc [hunk 5]
```diff
@@ -89,8 +90,8 @@ mvn verify
 {: codeblock}
 endif::[]
 
-This command might take some time to run the first time because the dependencies and the Docker image for Open Liberty must download. If you 
-run the same command again, it will be faster.
+This command might take some time to run the first time because the dependencies and the Docker image for Open Liberty must download.
```
Comment (Rwalls1) [inline README.adoc]: Change "the first time" to "initially" to be more concise.

File: README.adoc [hunk 6]
```diff
@@ -89,8 +90,8 @@ mvn verify
 {: codeblock}
 endif::[]
 
-This command might take some time to run the first time because the dependencies and the Docker image for Open Liberty must download. If you 
-run the same command again, it will be faster.
+This command might take some time to run the first time because the dependencies and the Docker image for Open Liberty must download. 
+If you run the same command again, it will be faster.
 
 The previous example shows how you can run integration tests from a cold start. With Open Liberty development mode, you can use MicroShed Testing to run tests on
 an already running Open Liberty server. Run the following Maven goal to start Open Liberty in development mode:
```
Comment (Rwalls1) [inline README.adoc]: Change "on an already running Open Liberty server" to "on an active Open Liberty server" to be more concise.

File: README.adoc [hunk 7]
```diff
@@ -154,54 +157,35 @@ Wait for the `Press the Enter key to run tests on demand.` message, and then pre
...
-include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.1.java[]
-----
-// if use PersonServiceIT.2.java, use following:
-//[source, java, linenums, role='code_column hide_tags=copyright,importInject,importMPApp,importAssertNotNull,importSharedContainerConfig,sharedContainerConfig,inject,personSvc,mpApp,container,testCreatePerson']
-//----
-//include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.2.java[]
-//----
-
-Next, the `PersonServiceIT` class outlines some basic information that informs how MicroShed Testing starts the application runtime and at which URL path the application will be available:
+The `PersonServiceIT` class outlines some basic information that informs how MicroShed Testing starts the application runtime and at which URL path the application will be available:
```
Comment (Rwalls1) [inline README.adoc]: Change "at which URL path the application will be available:" to "at which URL path the application is available".

File: README.adoc (hunk 2 above)
Comment (Rwalls1) [inline README.adoc]: Add an identifying noun after "HTTP 200" to indicate what is being referred to, so change to something like "the HTTP 200 message".

File: README.adoc [hunk 8]
```diff
@@ -154,54 +157,35 @@ Wait for the `Press the Enter key to run tests on demand.` message, and then pre
...
 ----
-[role="edit_command_text"]
-Import the [hotspot=importMPApp file=1]`ApplicationContainer` class and the [hotspot=importContainer file=1]`Container` annotation, create the [hotspot=mpApp file=1]`ApplicationContainer` application, and annotate the application with [hotspot=container file=1]`@Container`.
 
 PersonServiceIT.java
 [source, java, linenums, role='code_column hide_tags=copyright,importInject,importAssertNotNull,importSharedContainerConfig,sharedContainerConfig,inject,personSvc,testCreatePerson']
 ----
 include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.2.java[]
 ----
+Import the [hotspot=importMPApp file=0]`ApplicationContainer` class and the [hotspot=importContainer file=0]`Container` annotation, create the [hotspot=mpApp file=0]`ApplicationContainer` application, and annotate the application with [hotspot=container file=0]`@Container`.
```
Comment (Rwalls1) [inline README.adoc]: Add an identifying noun after "[hotspot=container file=0]`@Container`" to clarify what's being referred to, so change to something like "the [hotspot=container file=0]`@Container` annotation".

File: README.adoc [hunk 9]
```diff
@@ -233,7 +216,8 @@ In this example, the [hotspot=personSvc file=0]`PersonService` injected type is
 [hotspot file=1]`io.openliberty.guides.testing.PersonService` class that is used in your application. 
 However, the _instance_ that gets injected is a REST client proxy. 
 So, if you call `personSvc.createPerson("Bob", 42)`, the REST client makes an HTTP POST request to the application 
-that is running at `\http://localhost:9080/guide-microshed-testing/people`, which triggers the corresponding Java method in the application.
+that is running at `\http://localhost:9080/guide-microshed-testing/people`,
```
Comment (Rwalls1) [inline README.adoc]: Add an identifying noun after "`\http://localhost:9080/guide-microshed-testing/people`".

**Review by Rwalls1 — COMMENTED** [review]
The updates look good, I just have a few comments.

**Review by Rwalls1 — APPROVED** [review]
The latest updates look good.

**Review by AustinSeto — APPROVED** [review]
No comment from ID on "dev/test" wording, so I'll approve

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**